### ISSUE TITLE
MVNP doesn't do well with AncientGates

### ISSUE THREAD

**Issue by Maplencheerio** [Multiverse/Multiverse-NetherPortals#162]
Whenever you create a gate with AncientGates and try to enter it it tells you that "there is no nether linked to (world)_Nether" 
I need both plugins for the server so im not sure what to do!

**Comment by nicegamer7** [comment]
I've never used AncientGates so I'm not too sure, but I think it can be replaced with MV Portals. From what I can tell, MV Portals has the same features, but if not feel free to let me know.

**Comment by Maplencheerio** [comment]
> I've never used AncientGates so I'm not too sure, but I think it can be replaced with MV Portals. From what I can tell, MV Portals has the same features, but if not feel free to let me know.

Yeah MV Portals doesn't have the same features and it has no migration so having to remake all the portals i have would be an issue

**Comment by nicegamer7** [comment]
In that case, I will see to make it compatible. Also, what are some the features missing from MV Portals?

**Comment by Maplencheerio** [comment]
> In that case, I will see to make it compatible. Also, what are some the features missing from MV Portals?

Simplicity, and no migration method. i have about 40 portals on the server right now, and recreating all of those is going to be a pain, and ancient Gates is really easy to use and simple for me. Ancient also doesn't require WorldEdit in order for it to be used, even though that's not an issue for my server NOW, it was when we first needed gates.

**Comment by benwoo1110** [comment]
To solve such problems in general, I have added new api in v5.1 for custom portal plugin's to hook into Multiverse-NetherPortals and tell it let the custom portal plugin to handle. It actually has already been done for our own Multiverse-Portals, but i created a more generic interface where other plugins can also hook their own logic into it. See: #315
